ext/zend_test/test.c

```diff
@@ -373,6 +373,17 @@ static ZEND_FUNCTION(zend_test_parameter_with_attribute)
 	RETURN_LONG(1);
 }
 
+static ZEND_FUNCTION(zend_test_sensitive_parameter)
```

Comment:
IMO we don't really need this function anymore as we have lots of function to test the feature with

Reply:
Okay, I can drop the commit when I rebase onto the latest master.